website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -5,6 +5,8 @@ description: |-
   How to deploy HCP Boundary services and login for the first time.
 ---
 
+## Deploy HCP Boundary and Login
```

Comment:
Since this is the page title, we'll need it to be an `<h1>` element instead of an `<h2>`.

```suggestion
# Deploy HCP Boundary and Login
```